bin/sl-deploy.js

```diff
@@ -5,21 +5,23 @@
 var Parser = require('posix-getopt').BasicParser;
 var debug = require('debug')('strong-deploy');
 var defaults = require('strong-url-defaults');
-var getPackageInfo = require('../lib/package').getPackageInfo;
 var defaultPackagePath = require('../lib/package').getPackagePath;
 var deploy = require('../');
 var fs = require('fs');
+var getPackageInfo = require('../lib/package').getPackageInfo;
+var glb = require('strong-globalize');
 var maybeTunnel = require('strong-tunnel');
 var path = require('path');
 
 function printHelp($0, prn) {
-  var USAGE = fs.readFileSync(require.resolve('./sl-deploy.txt'), 'utf-8')
-      .replace(/%MAIN%/g, $0)
-      .trim();
-
+  var USAGE = glb.t('sl-deploy.txt')
```

Comment:
In this scenario it totally makes sense, but there's something weird to me about how it completely abstracts away the file opening and reading while still clearly operating on a specified file. I would expect there to have been a path resolver wrapped around the path string instead. I imagine it to look like this:
```js
var USAGE = fs.readFileSync(glb.resolve('./sl-deploy.txt'), 'utf-8')
    .replace(/%MAIN%/g, $0)
    .trim();
```

If the logical resource is simply a string and the fact that it is in a file is an implementation detail, then I would expect the name to be something like `'USAGE'` or something.